banman/store.go

```diff
@@ -66,6 +66,9 @@ type Store interface {
 
 	// Status returns the ban status for a given IP network.
 	Status(*net.IPNet) (Status, error)
+
```

Comment:
In commmit message: `banmen` -> `banman`

Reply:
Oh okay thanks.